### PR TITLE
BXMSPROD-911 get source repository name from forked project list

### PR DESCRIPTION
**Pull request by Ginxo**
https://issues.redhat.com/browse/BXMSPROD-911
Changes tested in https://rhba-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/KIE/job/master/job/pullrequest/job/new-PR-lienzo-tests-master.pullrequests/16
Project https://github.com/Ginxo/lienzo-tests renamed temporary as https://github.com/Ginxo/lienzo-tests-renamed-for-911

```
13:12:33  + git pull 'https://****@github.com/Ginxo/lienzo-tests-renamed-for-911' BXMSPROD-911_test
13:12:17   > git config core.sparsecheckout # timeout=10
13:12:17   > git checkout -f 1be66027322fbea828c2f2092a1444e1e3668297 # timeout=10
13:12:17   > git rev-list --no-walk 1be66027322fbea828c2f2092a1444e1e3668297 # timeout=10
13:12:33  From https://github.com/Ginxo/lienzo-tests-renamed-for-911
13:12:33   * branch            BXMSPROD-911_test -> FETCH_HEAD
13:12:33  Updating 1be6602..a986a53
13:12:33  Fast-forward
13:12:33   README.md | 1 +
13:12:33   1 file changed, 1 insertion(+)
[Pipeline] }
[Pipeline] // withCredentials
[Pipeline] sh
13:12:33  + git log --oneline -1
[Pipeline] echo
13:12:33  
13:12:33      -------------------------------------------------------------
13:12:33      [INFO] Source merged into Target
13:12:33      -------------------------------------------------------------
13:12:33      Target: 1be6602 BXMSPROD-912 openjdk11 build execution added to pull_request.yml (#98)
13:12:33      Produced: a986a53 test
13:12:33      -------------------------------------------------------------
13:12:33   
```

### CODE REVIEW

**Comment by Ginxo** [comment]
renamed back to https://github.com/Ginxo/lienzo-tests

**Comment by radtriste** [comment]
Could not find it in Kogito pipelines

File: vars/githubscm.groovy [hunk 1]
```diff
@@ -44,21 +45,21 @@ def getRepositoryScm(String repository, String author, String branches, String c
     return repositoryScm
 }
 
-def mergeSourceIntoTarget(String repository, String sourceAuthor, String sourceBranches, String targetAuthor, String targetBranches, String credentialId = 'kie-ci') {
-    println "[INFO] Merging source [${sourceAuthor}/${repository}:${sourceBranches}] into target [${targetAuthor}/${repository}:${targetBranches}]..."
-    checkout(resolveRepository(repository, targetAuthor, targetBranches, false, credentialId))
+def mergeSourceIntoTarget(String sourceRepository, String sourceAuthor, String sourceBranches, String targetRepository, String targetAuthor, String targetBranches, String credentialId = 'kie-ci') {
+    println "[INFO] Merging source [${sourceAuthor}/${targetRepository}:${sourceBranches}] into target [${targetAuthor}/${targetRepository}:${targetBranches}]..."
```
Comment (radtriste) [inline vars/githubscm.groovy:49]: Shouldn't it be ?

```suggestion
    println "[INFO] Merging source [${sourceAuthor}/${sourceRepository}:${sourceBranches}] into target [${targetAuthor}/${targetRepository}:${targetBranches}]..."
```

**Comment by Ginxo** [comment]
thanks @radtriste

**Pull request merged**